src/main/kotlin/com/msd/robot/domain/Robot.kt

```diff
@@ -191,4 +191,10 @@ class Robot(
             energyRegen
         }
     }
+
+    fun regenerateHealth(amount: Int) {
```

Comment:
According to the [wiki](https://the-microservice-dungeon.github.io/docs/rules/trading), robots can only be repaired to max health. Also please document your methods.

Reply:
This method should also probably be renamed to `repairRobot`.